config/shairport_metadata.bash

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
+# Clear out any previous album cover images, then navigate to the proper directory #
```

Comment:
add a guard here before the rm -rf to validate the $1 arg, maybe something like this:
```
src="$1"
iif ((src <= 0 || src > 4)); then
  exit 1;
fi

Reply:
This way we cant delete random stuff even if this script is called in a weird way